firstvoices/backend/views/contact_us_views.py

```diff
@@ -0,0 +1,174 @@
+        """
+
+        site = self.get_validated_site().first()
+        perm = Site.get_perm("change")
+        if not request.user.has_perm(perm, site):
+            return Response({"message": error_403}, status=403)
+        return super().list(request, *args, **kwargs)
+
+    def create(self, request, *args, **kwargs):
+        """
+        Sends emails to the addresses listed on the site contact_us_email and contact_us_users fields. A list of
+        fallback email addresses can be set in the AppJson model with the key 'contact_us_default_emails'. If the
+        site does not have any contact emails or users, the fallback email will be used. Additionally, the site can
+        specify a list of excluded words in the AppJson model with the key 'contact_us_excluded_words'. If an email
+        contains any of these words in the email, name, or message it will not be sent.
+        """
+
+        logger = logging.getLogger(__name__)
+        site = self.get_validated_site().first()
+        if request.user.is_anonymous:
+            return Response(
+                {"message": "You must be logged in to send an email."}, status=403
+            )
+        try:
+            to_email_list = site.contact_email
+            to_email_list.extend(site.contact_users.values_list("email", flat=True))
+            name = request.data["name"]
+            from_email = request.data["email"]
+            message = request.data["message"]
+
+            # Validate that the email field from the form is a valid email address
+            validate_email(from_email)
+
+            # If the site does not have any contact emails or users, fallback to the default emails in AppJson
+            if to_email_list is None or len(to_email_list) == 0:
+                logger.warning(
+                    f'No emails found in the contact_email and contact_users fields found on site "{site.title}".'
+                )
+                to_email_list = get_fallback_emails()
+
+            # If the site has excluded words, validate that the message does not contain any of them
+            self.validate_no_excluded_words(message, name, from_email, site)
+
+            try:
+                # Format the final email
+                final_message = (
+                    "The following message was sent from the contact us form on the FirstVoices website:\n\n"
+                    f"Name: {name}\n"
+                    f"Email: {from_email}\n\n"
+                    f"Message:\n{message}\n"
+                )
+                send_mail(
+                    subject=f"Contact Us Form Submission from {name} ({from_email})",
+                    message=final_message,
+                    recipient_list=to_email_list,
+                    from_email=settings.DEFAULT_FROM_EMAIL,
+                )
+            except (ConnectionRefusedError, SMTPException) as e:
+                logger.error(f"Contact us email failed to send. Error: {e}")
```

Comment:
Please put the message into the exception and skip the error log. (Unhandled exceptions will be logged anyway.)